docs/source/release/v6.2.0/mantidworkbench.rst

```diff
@@ -5,57 +5,75 @@ Mantid Workbench Changes
+- Fixed the advanced plotting dialog incorrect layout, which caused the options to be partially occluded.
 - Scroll bars added to about dialog if screen resolution is too low.
 - Fixed missing 'on top' windowing behaviour for the matrix and table workspace data displays.
-- Sliceviewer now doesn't normalise basis vectors for HKL data such that Bragg peaks appear at integer HKL for cuts along e.g. HH0
 - Uninstalling from Windows "Apps & features" list will now run the uninstaller as the current Windows user and delete all shortcuts.
+- Added missing icon for the uninstaller in Windows "Apps & features" list.
 - Fixed a bug where parameters wouldn't update in the fit property browser when fitting a single function with ties.
+- Fixed a bug where the workspace index spinbox in the fit browser wouldn't update when the user added or removed curves from the figure.
 - Fixed a bug retrieving algorithm history from a workspace when the retrieval methods were chained together.
-- Added missing icon for the uninstaller in Windows "Apps & features" list.
 - Fixed a bug where output workspaces of different types would interfere with successive calls to binary operations, such as multiply.
-- Fixed JSON serialization issue of MantidAxType by explicitly extracting its value
-- Fixed a bug in the Sliceviewer when transposing MDE workspaces multiple times would cause the data to become all zeros.
-- Fixed a bug in colorfill plots which lead to the loss of a spectrum from the resulting image.
+- Fixed JSON serialization issue of MantidAxType by explicitly extracting its value.
 - Fixed a bug where the errorbar tab in the figure options was wrongly enabled while selecting multiple curves.
-- Fixed a bug where removing the plot guess line in the fit browser could lead to an exception being thrown.
 - Fixed a bug where marker formatting options were disabled upon opening the figure options.
-- Fixed a bug where the workspace index spinbox in the fit browser wouldn't update when the user added or removed curves from the figure.
-- Fixed out of range errors in the Sliceviewer that sometimes occured whilst hovering over transposed data.
 - Fixed the help icon not showing on OSX and high-resolution monitors.
 - Tabbing between fields in the error reporter now works as expected, rather than jumping to a random place each time.
-- Fixed the advanced plotting dialog incorrectly laying out, causing the options to be partially occluded.
-- Fixed a bug in the Instrument Viewer causing the projection to not be updated when different axis views were selected in Full 3D.
+- Fixed a bug in the :ref:`InstrumentViewer` causing the projection to not be updated when different axis views were selected in Full 3D.
+
+Sliceviewer
+-----------
 
+Improvements
+############
+
+- Peaks can now be added or removed from a PeaksWorkspace using the :ref:`peaks overlay <sliceviewer_peaks_overlay>` in :ref:`sliceviewer`.
```

Comment:
Thanks for grouping these, however, now it seems redundant to repeat :ref:`sliceviewer` in each item as they are all under one heading. Perhaps sublists would be better. The topmost item can say `Several improvements have been done in :ref:`sliceviewer`. Then the sub-items can just enumerate them. Peaks overlay can, of course, be referenced as it is a different link.